packages/next/src/lib/metadata/resolve-metadata.ts

```diff
@@ -331,9 +330,6 @@ async function getDefinedViewport(
   props: any,
   tracingProps: { route: string }
 ): Promise<Viewport | ViewportResolver | null> {
-  if (isClientReference(mod)) {
```

Comment:
We actually don't need this as the module is transformed, there's no client components. The blow checking is enough